engine/CMakeLists.txt

```diff
@@ -1189,6 +1210,7 @@ ELSEIF (CMAKE_SYSTEM_NAME STREQUAL Linux)
     IF(HAS_APT)
         MESSAGE("-- Configuring Debian Packaging")
         # See https://cmake.org/cmake/help/v3.3/module/CPackDeb.html
+        SET(CPACK_DEBIAN_PACKAGE_NAME "Vega-Strike")
```

Comment:
Did we miss this in master in the recent changes?

Reply:
So we were already setting `CPACK_PACKAGE_NAME`. And `CPACK_DEBIAN_PACKAGE_NAME` was supposed to inherit its default setting from the former. But that didn't seem to work, at least on one of the distros I tried. That part of the filename was missing. So I added this line in to set `CPACK_DEBIAN_PACKAGE_NAME` explicitly.

Reply:
Thanks!